build_deploy.sh

```diff
@@ -1,13 +1,15 @@
 #!/bin/bash
 
 TAG=$(git rev-parse --short=7 HEAD)
-
 BM_INVENTORY_IMAGE="quay.io/app-sre/bm-inventory"
+BUILD_FOLDER=$PWD/build
+GIT_REVISION=$(git rev-parse HEAD)
 
-docker build -t "${BM_INVENTORY_IMAGE}:latest" -f Dockerfile.demo .
+skipper make create-build-dir
+CGO_ENABLED=0 go build -o $BUILD_FOLDER/bm-inventory cmd/main.go
+skipper make create-python-client
+docker build -t "${BM_INVENTORY_IMAGE}:latest" --build-arg GIT_REVISION=$GIT_REVISION -f Dockerfile.bm-inventory .
```

Comment:
not crucial, but to keep consistency with the Makefile, you can do something similar to 'update' rule.
i.e. 
SERVICE=${BM_INVENTORY_IMAGE}:latest
GIT_REVISION=${GIT_REVISION} docker build --build-arg GIT_REVISION -f Dockerfile.bm-inventory . -t $(SERVICE)